### PR TITLE
Change drawer position to relative

### PR DESCRIPTION
**Pull request by MicheleK17**
Ticket: #388
Change-Id: If90433757336eef261c783a496c4fb96dd878227

### CODE REVIEW

File: src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx [hunk 1]
```diff
@@ -49,32 +49,38 @@ const useStyles = makeStyles(() => ss);
...
+    return (
+        <Drawer
+            className={classes.drawer}
+            variant="persistent"
+            anchor="left"
+            open={props.isOpen}
+            classes={{
+                paper: classes.drawerPaper,
+            }}
+            sx={{
```
Comment (GerberI) [inline src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx:72]: I don't understand this, looks like a hack. The cause of the problem might be elsewhere.
But for a fix it is ok.

File: src/app/Shared/ClearSelectorButton.component.tsx [hunk 2]
```diff
@@ -47,18 +47,21 @@ export function ClearSelectorComponent(props: {
...
-                    <CancelIcon
-                        css={css`
-                            height: 20px;
-                            width: 20px;
-                        `}
-                    />
-                </IconButton>
+                <span>
+                    <IconButton
+                        css={iconButtonStyle}
```
Comment (GerberI) [inline src/app/Shared/ClearSelectorButton.component.tsx:52]: iconButtonStyle: display: inline-block or so could have worked too.

**Review by GerberI — APPROVED** [review]

File: src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx (hunk 1 above)
Comment (MicheleK17) [reply src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx:72]: No this is not a hack. This is new by the update to material ui 5. In the long run the whole css will be changed to it.

File: src/app/Shared/ClearSelectorButton.component.tsx (hunk 2 above)
Comment (MicheleK17) [reply src/app/Shared/ClearSelectorButton.component.tsx:52]: This is the official solution that Material Ui recommends.

Comment (GerberI) [reply src/app/Shared/ClearSelectorButton.component.tsx:52]: 👍

File: src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx (hunk 1 above)
Comment (GerberI) [reply src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx:72]: And what does sx mean?
I think they want to make their components more customizable. Or what is the reason behind it?

**Pull request merged**